zenodo/modules/records/templates/zenodo_records/macros/admin.html

```diff
@@ -0,0 +1,109 @@
+{#
+# This file is part of Zenodo.
+# Copyright (C) 2015 CERN.
+#
+# Zenodo is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+#}
+
+{%- macro user_info_table(user) %}
+<div class="list-group-item col-sm-6">
+    <strong>ID:</strong> {{user.id}}
+</div>
+<div class="list-group-item col-sm-6">
+    <strong>Email:</strong> {{user.email}}
+</div>
+<div class="list-group-item col-sm-6">
+  <strong>Username:</strong> {{user.profile and user.profile.username}}
+</div>
+<div class="list-group-item col-sm-6">
+  <strong>Full name:</strong> {{user.profile and user.profile.full_name}}
+</div>
+<div class="list-group-item col-sm-6">
+  <strong>Active?:</strong> {{user.profile and user.active}}
+</div>
+<div class="list-group-item col-sm-6">
+  <strong>Confirmation Date:</strong> {{user.confirmed_at}}
+</div>
+<div class="list-group-item col-sm-6">
+  <strong>Last login:</strong> {{user.last_login_at}}
+</div>
+<div class="list-group-item col-sm-6">
+  <strong>Current login:</strong> {{user.current_login_at}}
+</div>
+<div class="list-group-item col-sm-6">
+  <strong>Last login IP:</strong> {{user.last_login_ip}}
+</div>
+<div class="list-group-item col-sm-6">
+  <strong>Current login IP:</strong> {{user.current_login_ip}}
+</div>
+<div class="list-group-item col-sm-6">
+  <strong>Login count:</strong> {{user.login_count}}
+</div>
+{%- endmacro %}
+
+{%- macro bucket_list(record_type) %}
+<tr>
+  <th>Object</th>
+  <th>URI</th>
+  <th>Size</th>
+  <th>Checksum</th>
+</tr>
+{%- for f in record_type.files %}
+<tr>
+  <td>{{f.key}}</td>
+  <td>{{f.file.uri}}</td>
+  <td>{{f.file.size}}</td>
+  <td>{{f.file.checksum}}</td>
+</tr>
+{%- endfor %}
+{%- endmacro %}
+
+{%- macro instance_titles() %}
+<th>ID</th>
+<th>Revision</th>
+<th>Creation Date</th>
+<th>Updated</th>
+{%- endmacro %}
+
+{%- macro instance_information(record_type) %}
+<tr>
+  <td>{{record_type.id}}</td>
+  <td>{{record_type.revision_id}}</td>
+  <td>{{record_type.created}}</td>
+  <td>{{record_type.updated}}</td>
+</tr>
+{%- endmacro %}
+
+{%- macro record_info_table(record_type, owners) %}
```

Comment:
```suggestion
{%- macro record_info_table(record) %}
```

- `record_type` sounds like a variable that would contain a string (e.g. 'image', 'journal-article', ...) or class (`ZenodoDeposit`, `ZenodoRecord`, ...) for a record. `record` in this case would still be true (since deposits are also sub-classes of record)
- `owners` is not used anywhere (remove from the macro calls as well)